test/discovery.js

```diff
@@ -188,16 +188,18 @@ describe("Crawler link discovery", function() {
     });
 
     it("should find resources in srcset attributes", function() {
-
+        // http://w3c.github.io/html/semantics-embedded-content.html#element-attrdef-img-srcset
         var links =
-            discover("<img src='pic-200.png' srcset='pic-200.png 200px, pic-400.png 400w'>", {
+            discover("<img src='pic.png' srcset='pic-200.png, pic-400.png 400w, pic-800.png 800w 2x'>", {
```

Comment:
Could we remove `800w` towards the end of this line? As we discussed previously, I don't think the spec permits two descriptors there, so it would be better to not make that part of the test suite (even if the new implementation would unofficially support that syntax)